recipes/sasview/meta.yaml

```
@@ -0,0 +1,59 @@
+{% set name = "sasview" %}
+{% set version = "v5.0_beta.2_8_g390494d7c" %}  # git describe --tags --dirty --always | sed 's/-/_/g'
+{% set file_ext = "tar.gz" %}
+
+package:
+  name: '{{ name|lower }}'
+  version: '{{ version }}'
+
+source:
+  git_url: https://github.com/SasView/sasview
+  git_rev: 390494d7cc6099973fce721a89afc666ae254c11
+
+build:
+  number: 0
```

If it should support Windows you need to add a `bld.bat` file else you need to explicitly skip it:
```suggestion
  number: 0
  skip: True  # [win]
```

---

Thank you!